doc/source/tutorial/tutorial_presenting_larray_objects.ipyml

```diff
@@ -364,12 +369,6 @@ cells:
     gender = Axis("gender=Male,Female")
     time = Axis("time=2013..2017")
     
-    # create and populate a new session in one step
-    # Python <= 3.5
-    demography_session = Session([('gender', gender), ('time', time), ('population', zeros((gender, time))), 
```

Comment:
Note that this syntax is also useful when you have objects with names which are not valid Python identifiers (such as labels starting with a number). This probably needs to be mentioned somewhere, but probably not here.

Reply:
Using a dictionary instead of a list of pairs would be clearer. Anyway, it is difficult and maybe not a good idea to show any pitfall or very specific cases in the tutorial. IMO, the tutorial is big enough right now to not add (very) specific cases (at least this one).

Reply:
Indeed a dict is better for Python3.7+, my bad. My comment was only valid for 3.6, but since we are doing away with it too, you can indeed simply remove that. Concerning special/tricky cases I think a tutorial is meant precisely to show those which users are likely to stumble into and how to "work around" them. In this case, I don't know how likely it is. I have seen array labels starting with numbers quite a few time, but never in a session, so removing the block is fine I guess.

Reply:
Your comment above makes me think that I may add something about how to create a group mixing separated labels and a slice (using Group.union()). What do you think ?

Reply:
A good idea (unless you plan to tackle #429 "soonish", **in which case** it would be better to tackle it first to show the nice syntax directly) but I don't know what's the best place to add that...